### DISCUSSION TITLE
Categories in a Statamic native way

### DISCUSSION THREAD

**Discussion by ghost** [jonassiewertsen/statamic-butik#175] [Show and tell]
Categories hold many products but sometimes need a basic description. 

It is disparate to have description of category in design files only.

This would be good for setting global category descriptions to be called in antlers for easily making Category cards similar to product cards

**Comment by jonassiewertsen** [comment]
I do agree. This seems to be related to this issue: https://github.com/jonassiewertsen/statamic-butik/issues/159

For the future I am thinking if it would be smartest do use Taxonomies for categories instead. Would this make sense? That approach would be closer to the `Statamic way`

**Reply by jonassiewertsen** [reply]
Taxonomies aren't that flexible at the moment. it's especially hard with translations. 

I am considering the use of a category collection instead:
- Good translations
- custom routes for your categories
- Editable fields as you need them
- Custom templates as you need them

**Reply by ghost** [reply]
This would work I think, but IMO it would be convenient to allow creation of the category collection content from in the create product blueprint.

> Taxonomies aren't that flexible at the moment. it's especially hard with translations.
> 
> I am considering the use of a category collection instead:
> 
>     * Good translations
> 
>     * custom routes for your categories
> 
>     * Editable fields as you need them
> 
>     * Custom templates as you need them

**Reply by leganz** [reply]
I think a category collection would be the best solution.  But it would a breaking change right?

**Reply by jonassiewertsen** [reply]
This will get implemented in butik version 4.

**Comment by freshface** [comment]
Small question to the thread. 
Are categories localizable atm in boutik?

**Reply by jonassiewertsen** [reply]
Not at the moment. This change will be the next one.

if implemented, categories will be localizable.

**Reply by freshface** [reply]
Thx, any ETA?
Needs to be translatable for a project i am working on :)

**Comment by jonassiewertsen** [comment]
This feature has been implemented in version 4, which is still work in progress. 

Butik products will be statamic collections in the future and the actual categories will be removed in favour of native Statamic options. 
You can either create categories as Taxonomies or another Collection. 

This change does give you full control of categories, as you need and like them. 
More detailed information will be added to the version 4 docs later.
